### PR TITLE
Update README.md to include Statuspage link

### PR DESCRIPTION
**Pull request by ryanlink**
This adds a paragraph to the bottom of `vets-api-clients` README.md with a link to Lighthouse Statuspage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,6 @@ Note that the Health (FHIR / Argonaut) APIs for access to Veteran health records
 
 ## Further Assistance
 If you have any questions or need further assistance, please email our team at api@va.gov. You will receive a human reply within 24 hours.  Additionally, you may open a comment or support request [inside this repo](https://github.com/department-of-veterans-affairs/vets-api-clients/issues/new/choose).
+
+## Incident and Outage Awareness (Statuspage)
+Please visit and subscribe to alerts from the [VA Lighthouse Statuspage](https://valighthouse.statuspage.io/) for up-to-the minute reports on API and gateway functionality.  Currently monitoring the dev environment only, with production coming soon.
```
Comment (jeff2d2) [inline README.md]:
```suggestion
Please visit and subscribe to alerts from the [VA Lighthouse Statuspage](https://valighthouse.statuspage.io/) for up-to-the minute reports on API and gateway functionality. Currently monitoring the dev environment only, with production coming soon.
```

**Comment by ryanlink** [comment]
I've removed the offending double space.  😂 please re-approve @jeff2d2

**Comment by jeff2d2** [comment]
DId you still want the paragraph above as is, where we ask them to send an email directly or file an issue on GH, but we don't send them to the Support page (where there's FAQs that might address what they need)?

**Comment by ryanlink** [comment]
Woof.  That would be good to update as well.  Momentito

**Comment by stephaniemace** [comment]
3 suggestions for your consideration:

- Update the intro from July 2019 to September 2019

- You could borrow Statuspage content from the September newsletter if you want to add more detail.  

Specifically, I would like to incorporate this language:
 **VA currently offers real-time information for our development environments. We are still working on plans to monitor production and staging environments.** 

Statuspage content from September 2019 newsletter
> When a system isn’t working, you need to know. We recently launched the new VA Lighthouse Statuspage and subscribe feature to help Developers and API consumers find out whether things are working as expected or if there’s an incident. Our VA API status page dashboard displays current and historical performance information. 
> 
> We encourage you to subscribe to receive API incident notifications via email. Simply click the “Subscribe To Updates” button and enter your preferred email address. If there is an incident, you’ll receive an email so it will be easy to learn more about the issue and when it has been resolved. You’ll also have the ability to manage your subscription and opt out of receiving notifications at any time. 
> 
> This is a valuable tool for our users to check system reliability and receive updates if anything changes. VA currently offers real-time information for our development environments. We are still working on plans to monitor production and staging environments. Our team will send further updates when additional features are available on the API status page. 

- Would it be appropriate to also offer a subscription link to the newsletter to help others keep in touch? My suggestion is to create a "Learn More" section at the very end and you could use this approved talking point:

Subscribe to the VA Lighthouse newsletter to receive regular program updates: https://public.govdelivery.com/accounts/USVAOIT/subscribers/new?preferences=true.

**Comment by ryanlink** [comment]
I like all of that, with the caveat that the format of this README might not lend itself to much more detail on Statuspage.  Let me put together another PR right quick!

**Comment by ryanlink** [comment]
Careful with that last URL @stephaniemace - it doesn't specify the Lighthouse topic.  That might be signing up for a general OIT newsletter.  Should be https://public.govdelivery.com/accounts/USVAOIT/subscriber/new?topic_id=USVAOIT_20

**Pull request merged**